src/transformState.js

```diff
@@ -5,7 +5,36 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const currentAction of actions) {
+    switch (currentAction.type) {
+      case 'addProperties':
+        const extraData = currentAction.extraData;
```

Comment:
you can also use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment?retiredLocale=pl)